ros2topic/ros2topic/api/__init__.py

```diff
@@ -37,6 +38,26 @@ def unsigned_int(string):
     return value
 
 
+def nonnegative_int(inval):
```

Comment:
what's the difference with `unsigned_int`?
both are the same

Reply:
I just blindly moved things here. It is the same but the unsigned_int has the exception note you mentioned below.